.github/workflows/go.yaml

```diff
@@ -9,50 +9,38 @@ on:
 permissions: read-all
 
 jobs:
-  lint:
+  test:
     runs-on: ubuntu-latest
+    needs: go-versions
+    continue-on-error: true  # make sure to run all versions, even if one fails
+
+    strategy:
+      matrix:
+        go-version: ${{ fromJSON(needs.go-versions.outputs.matrix) }}
+
     steps:
-    - uses: actions/checkout@8e8c483db84b4bee98b60c0593521ed34d9990e8  # v6.0.1
+    - uses: "actions/checkout\
+             @8e8c483db84b4bee98b60c0593521ed34d9990e8"  # v6.0.1
       with:
         persist-credentials: false
-    - name: Set up Go
```

Comment:
@ingydotnet it's worth asking - why remove the third party approach to Golang version testing in favor of the custom version written in bash above?